packages/web/src/store/application/ui/userListModal/slice.ts

```diff
@@ -15,6 +15,7 @@ type SetUsersPayload = {
   userListType: UserListType
   entityType: UserListEntityType
   id: ID
+  entity?: any
```

Comment:
what's this for?

Reply:
super awkward but before we were passing notification id and selecting it from the store, now since we arent saving notifs we have to pass the whole thing to the user-list modal, which required adding a new thing to user-list action. I can be more explicit and say "notification" here and a comment so it's clear why this is here though

Reply:
ahhh okay sg